test/e2e/trafficroute/kuma_universal.go

```diff
@@ -0,0 +1,245 @@
+sources:
+  - match:
+      kuma.io/service: demo-client
+destinations:
+  - match:
+      kuma.io/service: echo-server_kuma-test_svc_8080
+conf:
+  loadBalancer:
+    roundRobin: {}
+  split:
+    - weight: 1
+      destination:
+        kuma.io/service: echo-server_kuma-test_svc_8080
+        version: v1
+    - weight: 1
+      destination:
+        kuma.io/service: echo-server_kuma-test_svc_8080
+        version: v2
+    - weight: 1
+      destination:
+        kuma.io/service: echo-server_kuma-test_svc_8080
+        version: v3
+`
+
+func KumaStandalone() {
+	const defaultMesh = "default"
+
+	var universal Cluster
+
+	BeforeEach(func() {
+		clusters, err := NewUniversalClusters([]string{Kuma3}, Verbose)
+		Expect(err).ToNot(HaveOccurred())
+
+		universal = clusters.GetCluster(Kuma3)
+
+		err = NewClusterSetup().
+			Install(Kuma(core.Standalone)).
+			Install(YamlUniversal(trafficRoute)).
+			Setup(universal)
+		Expect(err).ToNot(HaveOccurred())
+		err = universal.VerifyKuma()
+		Expect(err).ToNot(HaveOccurred())
+
+		echoServerToken, err := universal.GetKuma().GenerateDpToken(defaultMesh, "echo-server_kuma-test_svc_8080")
+		Expect(err).ToNot(HaveOccurred())
+		demoClientToken, err := universal.GetKuma().GenerateDpToken(defaultMesh, "demo-client")
+		Expect(err).ToNot(HaveOccurred())
+
+		err = NewClusterSetup().
+			Install(EchoServerUniversal("dp-echo-1", defaultMesh, "echo-universal-1", echoServerToken,
+				WithTransparentProxy(true),
+				WithServiceVersion("v1"))).
+			Install(EchoServerUniversal("dp-echo-2", defaultMesh, "echo-universal-2", echoServerToken,
+				WithTransparentProxy(true),
+				WithServiceVersion("v2"))).
+			Install(EchoServerUniversal("dp-echo-3", defaultMesh, "echo-universal-3", echoServerToken,
+				WithTransparentProxy(true),
+				WithServiceVersion("v3"))).
+			Install(DemoClientUniversal(AppModeDemoClient, defaultMesh, demoClientToken, WithTransparentProxy(true))).
+			Setup(universal)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		if ShouldSkipCleanup() {
+			return
+		}
+		err := universal.DeleteKuma()
+		Expect(err).ToNot(HaveOccurred())
+		err = universal.DismissCluster()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	It("should access all instances of the service", func() {
+		instances := map[string]bool{}
+		Eventually(func() bool {
+			stdout, _, err := universal.ExecWithRetries("", "", "demo-client",
+				"curl", "-v", "-m", "3", "--fail", "echo-server_kuma-test_svc_8080.mesh")
+			if err != nil {
+				return false
+			}
+
+			switch {
+			case strings.Contains(stdout, "echo-universal-1"):
+				instances["echo-universal-1"] = true
+			case strings.Contains(stdout, "echo-universal-2"):
+				instances["echo-universal-2"] = true
+			case strings.Contains(stdout, "echo-universal-3"):
+				instances["echo-universal-3"] = true
+			}
+			if len(instances) < 3 {
+				fmt.Printf("%v/3\n", len(instances))
+			}
+			return len(instances) == 3
+		}, "30s", "500ms").Should(BeTrue())
+	})
+}
```

Comment:
I think we can do better here. I'd like to have an easy way to add new test cases of TrafficRoute without spinning up new environment from the start and adding more and more time to already slow tests. Look at the TrafficPermission test that I recently introduced, you can add new `It()` with new test case, and the execution will take a couple of seconds tops.

What I'd like to see is that we can add other tests like 20% to v1, 80% to v2 or 100% to one version or 100% to completely different service.

Then, we could FINALLY remove minikube targets with a makefile e2e.mk mess.